service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -15,6 +15,13 @@ import (
 	"github.com/giantswarm/cluster-operator/service/controller/key"
 )
 
+type clusterProfile string
+
+const (
+	unknown clusterProfile = "UNK"
```

Comment:
Is there another abbreviation we can use? I don't think `UNK` is that widely used.

Also I'm not sure its unknown. It's just that XS is the only profile we are choosing to define so far.

Reply:
Was doubting, thought this was more explicit, compared to zero value / empty string. Will change to empty string as default here and in nginx app chart definition.

Reply:
Done, PTAL

Reply:
Thanks I think empty string as the default is good. 👍